frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseProjectDependentAction.java

```diff
@@ -8,18 +8,30 @@
 import org.checkerframework.checker.guieffect.qual.UIEffect;
 
 import com.virtuslab.branchlayout.api.readwrite.IBranchLayoutWriter;
-import com.virtuslab.gitmachete.frontend.actions.expectedkeys.IWithLogger;
 import com.virtuslab.gitmachete.frontend.ui.api.table.BaseEnhancedGraphTable;
 import com.virtuslab.gitmachete.frontend.ui.providerservice.BranchLayoutWriterProvider;
 import com.virtuslab.gitmachete.frontend.ui.providerservice.GraphTableProvider;
 import com.virtuslab.gitmachete.frontend.ui.providerservice.SelectedGitRepositoryProvider;
+import com.virtuslab.logger.IEnhancedLambdaLogger;
 
 public abstract class BaseProjectDependentAction extends DumbAwareAction implements IWithLogger {
+  @UIEffect
+  private boolean isUpdateInProgressOnUIThread;
+
+  @Override
+  public boolean isLoggingDiscouraged() {
```

Comment:
maybe just `isLoggingAllowed` and reverse the logic?

Reply:
Finally reversed the logic and changed to `isLoggingAcceptable`